pages/profile.tsx

```diff
@@ -1,103 +1,193 @@
-import React from "react";
+import { useQuery } from "@apollo/client";
+import React, { useState } from "react";
+import {
+  FetchUserProfileMetadataResponse,
+  FETCH_USER_PROFILE_METADATA,
+  UserProfileMetadata,
+} from "../graphql/fetchUserProfile";
+import { useAuth } from "../lib/authContext";
+
+// query to add User's name, username, email, created at date (joined), profile picture, achievements / badges?? prob existing quiz
 
 export default function Profile(props) {
+  const { user } = useAuth();
```

Comment:
what happens if you try to visit this page as a logged out user? 

You can protect this page from logged out users by adding this line to the bottom of the file: 

`Profile.auth = true;`

You can see how I protected the student portal page here: https://github.com/skillify-ca/skillify-web/blob/main/pages/studentPortal/%5Bslug%5D.tsx#L113